pxtlib/emitter/cloud.ts

```diff
@@ -89,8 +89,9 @@ namespace pxt.Cloud {
         })
     }
 
+    const isElectron = !!(window as any).pxtElectron || !!(window as any).ipcRenderer;
```

Comment:
there should be a help on for this

Reply:
It's in webapp/src/electron currently which is not accessible in here, should I move that around somewhere else? Would need more testing I would assume

Reply:
should be in browsersutil. let's minize casting window to any (and window might be missing in nodejs)

Reply:
Sounds good.